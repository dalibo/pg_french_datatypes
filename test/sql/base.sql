\set ECHO 0
\i sql/pg_french_datatypes.sql
\set ECHO all

CREATE TABLE test_pg_french_datatypes (
	c code_postal_fr,
	j jour,
	m mois,
	n numero_securite_sociale_fr
);

INSERT INTO test_pg_french_datatypes("c") 
VALUES (87120);

INSERT INTO test_pg_french_datatypes("c")
VALUES (75000000);

INSERT INTO test_pg_french_datatypes("n")
VALUES ('253072B07300483');

INSERT INTO test_pg_french_datatypes("n")
VALUES ('25307200730A483');

DROP TABLE test_pg_french_datatypes;
