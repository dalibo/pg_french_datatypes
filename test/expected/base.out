\set ECHO 0
CREATE TABLE test_pg_french_datatypes (
	c code_postal_fr,
	j jour,
	m mois,
	n numero_securite_sociale_fr
);
INSERT INTO test_pg_french_datatypes("c") 
VALUES (87120);
INSERT INTO test_pg_french_datatypes("c")
VALUES (75000000);
ERROR:  value for domain code_postal_fr violates check constraint "code_postal_fr_check"
INSERT INTO test_pg_french_datatypes("n")
VALUES ('253072B07300483');
INSERT INTO test_pg_french_datatypes("n")
VALUES ('25307200730A483');
ERROR:  value for domain numero_securite_sociale_fr violates check constraint "numero_securite_sociale_fr_check"
DROP TABLE test_pg_french_datatypes;
